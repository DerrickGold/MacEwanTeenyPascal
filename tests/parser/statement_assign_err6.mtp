begin
	(* Not a real value *)	
	something := 10something2;
end
.