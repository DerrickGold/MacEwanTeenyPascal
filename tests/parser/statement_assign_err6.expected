Syntax Error: line 3:
Unexpected token (TOK_ERROR, '10something2'), expected a token type of:  
  'TOK_LPAREN' or
  'TOK_KEY_NOT'
